README.md

```diff
@@ -48,19 +50,41 @@ Options:
+  generate-script       Generates a migration script. This provides you the ability to review the steps that this tool will take, and optionally modify the script if desired before running it.
+  grant-migrator-role   Allows an organization admin to grant a USER or TEAM the migrator role for a single GitHub organization. The migrator role allows the role assignee to perform migrations into the target organization.
+                        Note: Expects GH_PAT env variable to be set.
+  migrate-repo          Invokes the GitHub APIs to migrate the repo and all repo data.
+  revoke-migrator-role  Allows an organization admin to revoke the migrator role for a USER or TEAM for a single GitHub organization. This will remove their ability to run a migration into the target organization.
+                        Note: Expects GH_PAT env variable to be set.
+```
+
+To generate a migration script, you'll need to set `GH_PAT` as an environment variable for your destination and `GH_SOURCE_PAT` for your source location. 
+
+## GitHub Enterprise Server (GHES) to GitHub Migration Usage
+
+General usage will use the `generate-script` command to create a script that can be used to migrate all repositories from a GHES organization.
+
+Due to many GHES instances containing firewall rules restricting the direct API access that GEI needs to perform a migration, the GEI CLI uploads the migration's archive data to Azure Blob Storage as a step prior to starting the migration using GEI. This allows GEI to perform a migration without directly accessing the GHES instance. Because of this extra step, you'll pass in a connection string to an Azure Storage account, specified in the parameters below.
+
+### Command Line
+
+Generating a migration script for GHES is very similar to GitHub to GitHub migration scripts, but requires a couple extra parameters. You can run `gh gei generate-script --help` to learn about all of the options.
+
+The extra options you will need when migrating from GHES are:
+- `--ghes-api-url`
+- `--azure-storage-connection-string` - it's recommend to use quotes around this value since the connection string itself uses characters that might otherwise be interpreted by bash
+- `--no-ssl-verify` (optional)
+
+Here's an example:
+```
+gh gei generate-script --github-source-org "source-ghes-org" --github-target-org "target-ghec-org" --ghes-api-url "https://api.myghesinstance.com" --azure-storage-connection-string "DefaultEndpointsProtocol=https;AccountName=myazureaccount;AccountKey=myazurestoragekey;EndpointSuffix=core.windows.net" --no-ssl-verify
 ```
 
-To generate a script, you'll need to set an `GH_PAT` as an environment variable for your destination and `GH_SOURCE_PAT` for your source location. 
+This will generate a new file (named `./migrate.ps1` by default) which gives you an opportunity to review the steps GEI-CLI will take. Once ready, running the `./migrate.ps1` file (in PowerShell, for example) will kick off the migration!
```

Comment:
```suggestion
This will generate a new file (named `./migrate.ps1` by default) which gives you an opportunity to review the steps GEI-CLI will take. Once ready, running the `./migrate.ps1` file will kick off the migration!
```